policies/new_chapter_policy.md

```diff
@@ -0,0 +1,136 @@
+  should be part of a minimum requirement.
+  
+- We don't have clear definition of what  or why one would want to start a chapter.
+  Generally, there is expectation for ongoing and regular community event, instead of
+  a one time workshop.
+ 
+- Each PyLadies chapter uses resources in many forms: they might have Pro meetup
+  account from The PSF, they may have website hosted, they might require funding from The PSF.
+
+- PyLadies admins received numerous inquiries from community members regarding local
+  chapters that turned out to be inactive, taking up additional resource/volunteer time
+  from Global leadership and administrator team to address these inquiries. 
+ 
+ - By better vetting new chapters, we hope to reduce unnecessary administrative work
+   down the road, and better focus our efforts in helping active chapters.
+
+## 2. What does it mean to start a PyLadies chapter?
+
+We should define the purpose of local PyLadies chapter, and why a chapter would be necessary.
+
+- help build and grow local Python community, through meetups, or sharing info through mailing lists, twitter, etc.
+
+What are the roles of a PyLadies chapter leader?
+
+- share news from global PyLadies leadership to their members
+
+- organize at least one meetup per year (?)
+
+- not necessarily be the one creating content/giving talk/workshop. 
+
+- provide report once a year to global PyLadies leadership 
+
+- notify global PyLadies leadership of changes in organizers (e.g. if they have new organizers, or if an organizer stepped down)
+
+### 3. What are minimum responsibilities and expectation when someone requested a PyLadies chapter? 
+
+- at least 2 co-organizers
+
+- preferably run by women
+
+- must be local (city-based). e..g Vancouver instead of BC or Canada
+
+- namespace must be descriptive (`kuala-lumpur` instead of `kl`)?
+
+### 4. What kind of support can new chapter leaders expect from Global PyLadies leadership team?
+
+- as a chapter they can use PyLadies logo and name
+
+- they will have @pyladies.com email address
+
+- they can have pyladies subdomain
+
+- access to organizers channel in slack
+
+- access to organizers resources
+
+Other privileges:
+
+- a chapter gets one vote in PyLadies board elections
+
+- ???
+
+## 5. What questions do we ask in our form?
+
+- name, email address, gender pronouns of at least 2 organizers
+
+- desired namespace
+
+- their Python/community background
+
+- intent/ability to keep the community active. e.g. if they have thought of plan idea for a meetup within the next 1 year?
+
+- their slack.pyladies.com username
+
+- the chapter's primary spoken language(s)
+
+- GDPR 
+
+Note: Form questions are being worked in [Issue #12](https://github.com/pyladies/project-admin-logisitics/pull/12)
+
+## 6. How do we review new chapter requests?
+
+Prospective organizers will submit new chapter requests via a Google Form.
+
+PyLadies admin & logistics team members will review new chapter requests once
```

Comment:
It would be nice for submitted requests to have a timeline for review + reply ("We examine requests on the first week of every month and reply to requests within 5 days of review." Setting expectations when you apply into the great unknown is always a better experience.

Reply:
yes, we used to have a bot that autoreplies with these info, so they got immediate response. We will definitely activate it again once new process in place.

So I think I will need to discuss with the team who will review incoming requests, on when/how often can they review chapter requests, and update the text here.

Reply:
I've added with timeline of how and when we review chapters. Please check if this is ok?